Shared/SentryIntegration.swift

```diff
@@ -5,8 +5,18 @@
 import Foundation
 import Sentry
 
-public class SentryIntegration {
-    public static let shared = SentryIntegration()
+public enum SentryTag: String {
+    case swiftData
```

Comment:
these are lowercase. But before they were uppercase. This might create some dupes?

Reply:
Let's just make them uppercase.